cves/CVE-2009-3548.yml

```diff
@@ -202,4 +219,11 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This appears to have been a design issue. The way the engineers designed the system, this vulnerability
+    was inherent. They didn't keep in mind the core security principle of secure by default. It wasn't until 
+    someone who was investigating security found this issue 3 years later that it was patched. It even mentioned 
+    in the response to the vulnerability report that the system behaved contrary to how it was described in the docs.
```

Comment:
Surprised this issue was present for 3 years. What do you think caused this issue to not be noticed for this length of time? I recall you mentioned this issue had no prior or post fix.